defaults/main.yml

```diff
@@ -177,3 +177,22 @@ cron_delete_old_users: "'hour': 0, 'minute': 10"
 cron_cleanup_stale_radacct: "'hour': 0, 'minute': 20"
 cron_delete_old_postauth: "'hour': 0, 'minute': 30"
 cron_delete_old_radacct: "'hour': 1, 'minute': 30"
+
+# cross-origin resource sharing (CORS) settings
+django_cors:
```

Comment:
I don't like that we are creating an abstractions for the settings are provided by django-cors-headers. I believe this is bloating the ansible role and making it complex for very small benefits. @nemesisdesign are you sure we want to proceed this way? 

If we really want to add variables to control these settings, we should add them only for the required settings (settings that do not have any default). For rest, we should ask users to add the django setting using the `openwisp2_extra_django_settings_instructions` variable.

Reply:
That could work if we explain it properly in the README.